appveyor.yml

```diff
@@ -56,12 +56,13 @@ install:
         Where-Object pullRequestId -eq $env:APPVEYOR_PULL_REQUEST_NUMBER)[0].buildNumber) { `
           throw "There are newer queued builds for this pull request, failing early." }
 
-    # We don't need to build the example recipe.
-    - cmd: if exist recipes\example rmdir recipes\example /s /q
-    # Skip if we are not in a PR.
-    - cmd: if "%APPVEYOR_REPO_NAME%" == "conda-forge/staged-recipes" if not defined APPVEYOR_PULL_REQUEST_NUMBER rmdir /q /s recipes
-    # Make sure the `recipes` directory exists if we removed it.
-    - cmd: if not exist recipes mkdir recipes
```

Comment:
Fixed effectively as we switched to using the cloned `master` in PR ( https://github.com/conda-forge/staged-recipes/pull/1467 ), which is pulled in around the same time the PR merge ref is pulled in. So won't be affected by these timing issues.

Also we don't run AppVeyor builds on `master` any more, but skip them outright thanks to PR ( https://github.com/conda-forge/staged-recipes/pull/1419 ). This makes the question raised before completely irrelevant.